server-spi-private/src/main/java/org/keycloak/events/EventType.java

```diff
@@ -63,6 +63,8 @@
     UPDATE_TOTP_ERROR(true),
     VERIFY_EMAIL(true),
     VERIFY_EMAIL_ERROR(true),
+    VERIFY_PROFILE(true),
+    VERIFY_PROFILE_ERROR(true),
```

Comment:
This is a bit weird - events should be create/update user, not verify user

Reply:
See https://issues.redhat.com/browse/KEYCLOAK-18429.